lib/routes/seagullApi.js

```diff
@@ -48,6 +49,106 @@ module.exports = function (crudHandler, userApiClient, metrics) {
     });
   }
 
+  function stringToBoolean(value) {
+    return _.includes(['true', 'yes', 'y', '1'], _.trim(value).toLowerCase());
+  }
+
+  function parseUsersQuery(req) {
+    var query = {};
+
+    var trustorPermissions = _.trim(req.params.trustorPermissions);
```

Comment:
just to refresh my memory - don't we also have custodial and and custodian concept too?